versioned/persist/rocks/src/main/java/org/projectnessie/versioned/persist/rocks/RocksDatabaseAdapter.java

```diff
@@ -95,6 +96,33 @@ public RocksDatabaseAdapter(
   public void eraseRepo() {
     try {
       db.delete(dbInstance.getCfGlobalPointer(), globalPointerKey());
+      Stream.of(
+              dbInstance.getCfGlobalPointer(),
+              dbInstance.getCfGlobalLog(),
+              dbInstance.getCfCommitLog(),
+              dbInstance.getCfRepoProps(),
+              dbInstance.getCfKeyList(),
+              dbInstance.getCfRefLog())
+          .forEach(
+              cf -> {
+                try (RocksIterator iter = db.newIterator(cf)) {
+                  List<ByteString> deletes = new ArrayList<>();
+                  for (iter.seekToFirst(); iter.isValid(); iter.next()) {
+                    ByteString key = ByteString.copyFrom(iter.key());
+                    if (key.startsWith(keyPrefix)) {
+                      deletes.add(key);
+                    }
+                  }
+                  deletes.forEach(
+                      key -> {
+                        try {
+                          db.delete(cf, key.toByteArray());
+                        } catch (RocksDBException e) {
+                          throw new RuntimeException(e);
+                        }
+                      });
```

Comment:
Could `deletes` be too large for in-memory processing? Should we perhaps do this in batches?.. or perhaps issue deletes inside the loop on like 110?

Reply:
It's code for tests only at the moment, so I'm not overly concerned here.